R/pkg/inst/tests/testthat/test_utils.R

```diff
@@ -17,6 +17,9 @@
 
 context("functions in utils.R")
 
+# Ensure Spark is installed
+sparkCheckInstall()
```

Comment:
Sure - that sounds fine. I was looking to see if `testthat` had any support for writing a `setup` that gets called before each test - Doesn't look like it has that